Vlad-Kireyev/Vlad-Kireyev.md

```diff
@@ -8,6 +8,12 @@ quote: "Logic will get you from A to B. Imagination will take you everywhere."
 quoteAuthor: "Albert Einstein"
 ---
 
+[[imgBadge]]
+| ![Azure_Fundamentals_Badge](../badges/Certification-microsoft-azure-developer-associate.png)
```

Comment:
Azure_Fundamentals_Badge is this right?

Reply:
Changed that to "Microsoft Certified: Azure Developer Associate", as it is on Bryden's profile